conans/client/tools/scm.py

```diff
@@ -203,7 +213,18 @@ def excluded_files(self):
         return excluded_list
 
     def get_remote_url(self):
-        return self.run("info --show-item url").strip()
+        url = self.run("info --show-item url").strip()
+        revision = self.run("info --show-item revision").strip()
+        return "{url}@{revision}".format(url=url, revision=revision)
+
+    def get_qualified_remote_url(self):
+        url = self.run("info --show-item url").strip()
+        revision = self.run("info --show-item revision").strip()
+        return "{url}@{revision}".format(url=url, revision=revision)
+        
+    def is_local_repository(self):
```

Comment:
We need this function in the `Git` class

Reply:
There is  already the function in the `Git` class.  This one is for SVN, so that the export class can print a warning message, if either Git or SVN uses a local repository.